### DISCUSSION TITLE
<title>Commit on github desktop </title>

### DISCUSSION THREAD

**Discussion by yobro2033** [community/community#45287] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hi, 

I just tried to commit from github desktop for large files, included over 30k files. Surprisingly it doesn't take long for it to be published, however, when I checked the commit from github Desktop, it returned 404 Not found, is it just because of the large files takes longer to show up?

Thanks

**Comment by jge162** [comment]
@yobro2033 Hello, it is possible that the large files are taking longer to show up. It is also possible that there were errors with the commit that were not visible in the GitHub Desktop app. You can check the status of the commit by going to the repository in a web browser and checking the "commits" tab. If the commit is not visible there, then it is likely that there was an error with the commit. This is just based on my experience. good luck...

**Reply by yobro2033** [reply]
There's no Commits tab on that repos. I'll prob wait a little bit longer but if it's not then prob there's an error when upload. Thanks!

**Reply by jge162** [reply]
yup just gotta wait at this point...hopefully it shows up...
